packages/default/scss/utils/_border.scss

```diff
@@ -0,0 +1,26 @@
+@include exports( "utils/border" ) {
+
+    // Border utility classes
+    $utils-border: (
+        "borders-all",
```

Comment:
I do not like the idea of `k-borders-all`. I believe `k-border` is okay. With this, we can remove the `if/else` logic.

@joneff @inikolova thoughts?

Reply:
I also do not like it :)
But we already discussed it with @joneff and agreed on the class name to be `k-borders-all`, since we already have `k-border` which can be seen [here](https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/dataviz/_layout.scss#L183).

Reply:
the border class in that particular snippet is no longer in use. i guess we can safely remove it